br/pkg/task/backup_raw.go

```diff
@@ -58,10 +58,10 @@ func DefineRawBackupFlags(command *cobra.Command) {
 }
 
 // CalcChecksumFromBackupMeta read the backup meta and return Checksum
-func CalcChecksumFromBackupMeta(ctx context.Context, curAPIVersion kvrpcpb.APIVersion, cfg *Config) (checksum.Checksum, []*utils.KeyRange, error) {
+func CalcChecksumFromBackupMeta(ctx context.Context, curAPIVersion kvrpcpb.APIVersion, cfg *Config) (*backuppb.BackupMeta, checksum.Checksum, []*utils.KeyRange, error) {
```

Comment:
Nit: maybe better to get `backupMeta` outside then pass in here. As what "CalcChecksumFromBackupMeta" says.

Reply:
done